src/components/WebexRemoteMedia/WebexRemoteMedia.jsx

```diff
@@ -62,9 +63,14 @@ export default function WebexRemoteMedia({className, meetingID, style}) {
 
   if (error) {
     remoteDisplay = (
-      <AlertBanner show type="warning">
-        Having trouble joining the meeting? Please check your connection.
-      </AlertBanner>
+      <>
+        <div className="icon-error">
```

Comment:
you don't need a container for the icon, it accepts a className prop:
```js
remoteDisplay = (
  <div class="media-error">
    <Icon name="error" className="media-error-icon" />
    <div>Having trouble</div>
    <div>Please</div>
  </div>
);
```
you can put the text css props on the `media-error` class and the icon color on the `media-error-icon` class